### DISCUSSION TITLE
Detect or find the scratch or dust on the image using Imagemagick

### DISCUSSION THREAD

**Discussion by atfsolace** [ImageMagick/ImageMagick#2502] [Help]
Hi ,

Version: ImageMagick 7.0.10-24 Q16

I want to detect or find the scratches on the images and find the count of scratches , and even also want to remove the scratches on image,  but not idea how to start with this.

I want show the message on UI if the images has scratches or dust.

Please help.

Thank you.

**Comment by snibgo** [comment]
What visually identifies a scratch or dust?

If you know that, you can count them, and use inpainting (aka hole-filling) to remove them, which really means guessing what those pixels should contain.

If you link to example images, we can give more specific advice.

**Reply by atfsolace** [reply]
Thank you for your reply.

I have lots of old cards images and actually I want to count the wrinkles and scratches on the images.

Please check the below some examples of cards images :
![sdfdsfdsf](https://user-images.githubusercontent.com/70207338/92115287-d09b9a00-ee0f-11ea-9654-06dff87e7cc1.jpg)
![51AQAT9cFiL _AC_](https://user-images.githubusercontent.com/70207338/92115300-d4c7b780-ee0f-11ea-9e6f-9fac8fa3da76.jpg)
